app/components/Calendar/index.js

```diff
@@ -32,21 +31,25 @@ import {
   DEFAULT_CALENDAR_VIEW,
 } from './calendarConstants';
 
+const ONE_HOUR_IN_MS = 3600000;
+
 moment.tz.setDefault(TIME_ZONE);
 // Re-apply moment-timezone default timezone, cause FullCalendar import have override the import
 
 class TimePickerCalendar extends Component {
   calendarRef = React.createRef();
+  timer = null;
 
   static propTypes = {
     date: PropTypes.string,
```

Comment:
```suggestion
  static propTypes = {
    date: PropTypes.instanceOf(Date),
```
I know you didn't touch this line on this PR, but I though I'd just point it out.